recipes/miniprot-boundary-scorer/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${PREFIX}/lib
+make CC="${CXX}"
```

Comment:
I am aware it can be confusing. This is a c++ package. It just happens to be that in the `Makefile` the variable used for the compiler is `CC` (CC=g++). So that is the variable we have to override.